### ISSUE TITLE
Dead Links

### ISSUE THREAD

**Issue by D4edalus** [HaziqRazali/Pedestrian-Intention-Prediction#3]
The following links in your Readme are dead

https://drive.google.com/open?id=13HPZdbdjg09paa-RMTpMPdm5nmgT1jYF
https://drive.google.com/file/d/1SWY2GDEyQp-wNmmFKMBTZHSYJN7HKfgU/view?usp=sharing

**Comment by Cat2794** [comment]
Yes, I have also tried accessing the two links from your Read Me, and unfortunately I encountered the 404 Not found error. If you could please update them I would highly appreciate it. Thank you.

**Comment by SujayNiranjan** [comment]
@Cat2794, @D4edalus did you guys get the link for the trained model?
